### ISSUE TITLE
dw_simple_sub.sh possibly broken

### ISSUE THREAD

**Issue by m3morin** [hpc/libhio#46]
this script does not appear to currently function -- possibly caused by the fact assumes moab (not current slurm) as scheduler which impacts multiple parts of script

**Comment by hjelmn** [comment]
Yeah. That script needs to be updated. I will try to update it this week.
